ambry-replication/src/test/java/com.github.ambry.replication/InMemoryStore.java

```diff
@@ -264,7 +264,7 @@ public void updateTtl(List<MessageInfo> infos) throws StoreException {
         }
         MessageFormatInputStream stream =
             new TtlUpdateMessageFormatInputStream(info.getStoreKey(), info.getAccountId(), info.getContainerId(),
-                info.getExpirationTimeInMs(), info.getOperationTimeMs());
+                info.getExpirationTimeInMs(), info.getOperationTimeMs(), info.getLifeVersion());
```

Comment:
Just want to make sure one thing. For V2 and V2, info.getLifeVersion() returns -1, right?

Reply:
it returns 0 here. 0 should be the lifeVersion to be persisted in the log.